functions.js

```diff
@@ -0,0 +1,50 @@
+const ourTest = {
```

Comment:
kindly practice good naming conversion `ourTest` could be something more professional.